### PR TITLE
tail: refresh page

### PR DESCRIPTION
**Pull request by EmilySeville7cfg**
- [ ] The page (if new), does not already exist in the repository.
- [x] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [x] The page has 8 or fewer examples.
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- [x] The page follows the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The page description includes a link to documentation or a homepage (if applicable).

**Version of the command being documented (if known):**

# ToDo

- [x] Use long options
- [x] Fix placeholders

### CODE REVIEW

**Review by mfrw — APPROVED** [review]
LGTM.
Thank you @EmilySeville7cfg

File: pages/common/tail.md [hunk 1]
```diff
@@ -5,24 +5,24 @@
...
 
 - Keep reading file until `Ctrl + C`:
 
-`tail -f {{file}}`
+`tail --follow {{path/to/file}}`
 
 - Keep reading file until `Ctrl + C`, even if the file is inaccessible:
 
-`tail -F {{file}}`
+`tail --follow={{path/to/file}} --retry`
```
Comment (navarroaxel) [inline pages/common/tail.md:24]: This "sounds" better, also the `=` makes some dark magic that I don't understand yet.
```suggestion
`tail --retry --follow {{path/to/file}}`
```
I don't know what this means, the `all.json` exists:
```bash
$ tail --retry --follow=all.json
tail: invalid argument ‘all.json’ for ‘--follow’
Valid arguments are:
  - ‘descriptor’
  - ‘name’
```
The command with `=` doesn't fail if the file is not created yet, and follow the file when it appears.

**Review by navarroaxel — CHANGES_REQUESTED** [review]

**Comment by sbrl** [comment]
If macOS doesn't support long form arguments, then we either need to clone the original page into `osx`, or keep using short form arguments on this page.

**Comment by EmilySeville7cfg** [comment]
Related issue: #7688.

File: pages/common/tail.md [hunk 2]
```diff
@@ -1,28 +1,29 @@
 # tail
 
 > Display the last part of a file.
+> See also: `head`.
 > More information: <https://www.gnu.org/software/coreutils/tail>.
 
 - Show last 'num' lines in file:
```
Comment (blueskyson) [inline pages/common/tail.md:7]:
```suggestion
- Show last 'count' lines in file:
```

File: pages/common/tail.md [hunk 3]
```diff
@@ -1,28 +1,29 @@
...
 
-`tail -n {{num}} {{file}}`
+`tail --lines {{count}} {{path/to/file}}`
 
 - Show all file since line 'num':
```
Comment (blueskyson) [inline pages/common/tail.md:11]:
```suggestion
- Show all file since line 'count':
```

File: pages/common/tail.md [hunk 4]
```diff
@@ -1,28 +1,29 @@
...
 
-`tail -n +{{num}} {{file}}`
+`tail --lines +{{count}} {{path/to/file}}`
 
 - Show last 'num' bytes in file:
```
Comment (blueskyson) [inline pages/common/tail.md:15]:
```suggestion
- Show last 'count' bytes in file:
```

File: pages/common/tail.md [hunk 5]
```diff
@@ -1,28 +1,29 @@
...
 > More information: <https://www.gnu.org/software/coreutils/tail>.
 
-- Show last 'num' lines in file:
+- Show last 'count' lines in file:
 
-`tail -n {{num}} {{file}}`
+`tail --lines {{count}} {{path/to/file}}`
 
-- Show all file since line 'num':
+- Show all file since line 'count':
```
Comment (navarroaxel) [inline pages/common/tail.md:11]: Perhaps this is better
```suggestion
- Print a file from a specific line number:
```

File: pages/common/tail.md [hunk 6]
```diff
@@ -1,28 +1,29 @@
...
 
-`tail -n +{{num}} {{file}}`
+`tail --lines +{{count}} {{path/to/file}}`
 
-- Show last 'num' bytes in file:
+- Show last 'count' bytes in file:
```
Comment (navarroaxel) [inline pages/common/tail.md:15]:
```suggestion
- Print a specific count of bytes from the end of a given file.
```

File: pages/common/tail.md [hunk 7]
```diff
@@ -1,28 +1,29 @@
...
 
-`tail -c {{num}} {{file}}`
+`tail --bytes {{count}} {{path/to/file}}`
 
 - Keep reading file until `Ctrl + C`:
```
Comment (navarroaxel) [inline pages/common/tail.md:19]: This is more specific
```suggestion
- Print the last lines of a given file and keep reading file until `Ctrl + C`:
```

**Review by navarroaxel — APPROVED** [review]

**Pull request merged**